### ISSUE TITLE
Building with no default features returns an erorr

### ISSUE THREAD

**Issue by andreeaflorescu** [rust-vmm/linux-loader#104]
Below is the error:

```
error[E0432]: unresolved import `crate::loader_gen::start_info`
  --> src/configurator/x86_64/pvh.rs:18:24
   |
18 | use crate::loader_gen::start_info::{hvm_memmap_table_entry, hvm_modlist_entry, hvm_start_info};
   |                        ^^^^^^^^^^ could not find `start_info` in `loader_gen`

error: aborting due to previous error; 1 warning emitted

For more information about this error, try `rustc --explain E0432`.
error: could not compile `linux-loader`

To learn more, run the command again with --verbose.
```

Using linux-loader with no other features is useful for when you want to consume just the cmdline loader and one of the exported loaders.

**Comment by uran0sH** [comment]
I will work on it.

**Comment by lauralt** [comment]
Fixed by #109
